app/styles/layout/l-homepage.scss

```diff
@@ -4,7 +4,6 @@
 .infomap {
   align-items: center;
   background-color: $color-8;
-  // background-image: url('../../assets/images/background_the_map.png');
   background-repeat: no-repeat;
   background-size: cover;
   display: flex;
```

Comment:
Makes any sense having background properties when we are not using a background anymore?